LUMO_toolbox/DOTHUB_LUMOdataQualityCheck.m

```diff
@@ -58,9 +64,91 @@ function DOTHUB_dataQualityCheck(nirsFileName,printFigFlag)
 dists = DOTHUB_getSDdists(SDtmp);
 
 % MOTION CHECK  #######################################################
-% - this script does not focus on motion content, but providing
+%timeSelectFlag allows the user to select a clean period (motion free) for analysis. 
+%A subset of channels are plotted to decrease computer load (this can
+%always be changed to plot all channels).
+if (timeSelectFlag==1) 
+    my_ans = 'n';
+    while my_ans == 'n';
+    fig1 = figure; set(fig1, 'units', 'normalized', 'outerposition', [0 0 0.8 0.8], 'color', [1 1 1])
+    plot(d(:,1500:2000)); xlim([0 size(d,1)]); ylim([0 1.5]); title('Select segment to include (start-end pair) - Press enter when finished', 'fontsize', 20)
+    xlabel ('Time (samples)','Fontsize', 15); ylabel('Intensity (raw)', 'Fontsize', 15)
+    set(gca, 'Yscale', 'log')
+    hold on
+% do data brushing manually
+    [x, ~] = ginput;
```

Comment:
Can you add a message to user (command line?) to tell them what they should do (click and drag data selection I guess?)

Reply:
Now reads: "%A subset of channels are plotted to decrease computer load (this can
%always be changed to plot all channels). % For manual time period selection, click once at the beginning of the segment to
% select the start point. Drag the cursor and click again to select the end
% point. Hit enter when finished. After, the figure will reappear with the
% selected time period highlighted in blue. This is to make sure that the correct period(s) was(were)
%selected beforre proceeding. If the period is correct, type 'y' and hit enter. "